src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -263,6 +263,10 @@ public partial class IPEndPoint : System.Net.EndPoint
         public override int GetHashCode() { throw null; }
         public override System.Net.SocketAddress Serialize() { throw null; }
         public override string ToString() { throw null; }
+        public static bool TryParse(string endPointString, out IPEndPoint result) { throw null; }
```

Comment:
Should the parameters be named `s` (here and in the implementation) to match what was approved in #31291 and most other `Parse`/`TryParse` methods?

Reply:
I originally used `s`, but throwing invalid parameter "`s`" on `Parse` didn't seem very informative. When I checked a few other classes to verify, I noticed that `IPAddress.Parse` used `ipString`. I'm open to whichever.

https://github.com/dotnet/corefx/blob/892d514664eb38274d48f5639ed8864ef0355a62/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L247

Reply:
@terrajobst, was argument name considered in the API review?